packages/tfm-ror51-rubygem-actioncable/tfm-ror51-rubygem-actioncable.spec

```diff
@@ -6,13 +6,18 @@
 
 Name:    %{?scl_prefix}rubygem-%{gem_name}
 Version: 5.1.4
-Release: 1%{?dist}
+Release: 2%{?dist}
 Summary: WebSocket framework for Rails
 Group:   Development/Languages
 License: MIT
 URL:     http://rubyonrails.org
 Source0: https://rubygems.org/gems/%{gem_name}-%{version}.gem
 
+Requires: %{?scl_prefix_ruby}ruby(release)
+Requires: %{?scl_prefix_ruby}ruby(rubygems)
+Requires: %{?scl_prefix}rubygem(nio4r)
+Requires: %{?scl_prefix}rubygem(websocket-driver)
+Requires: %{?scl_prefix}rubygem(actionpack) = %{version}
```

Comment:
Is there logic to replace this exact version with the macro?

Reply:
Logic? RPM should handle it for us.

Reply:
I mean that the gemspec might include version 5.1.4 but here you have the (better) macro.